src/core/pluginsmodel.cpp

```diff
@@ -0,0 +1,186 @@
+/***************************************************************************
+ pluginsmodel.cpp - PluginsModel
+
+ ---------------------
+ begin                : June 2025
+ copyright            : (C) 2025 by Mohsen Dehghanzadeh
+ email                : mohsen@opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "pluginsmodel.h"
+
+PluginsModel::PluginsModel( QObject *parent )
```

Comment:
PluginsModel -> PluginModel , we don't pluralize the noun here (e.g. BookmarkModel, DrawingTemplateModel, etc.). We do capitalize QFieldCloudProject>s<Model but that should eventually be fixed :)